php/exercicio10/exercicio10frontend.php

```diff
@@ -0,0 +1,63 @@
+<!--Entrar com 20 números e imprimir a soma
+dos positivos e o total de números negativos
+-->
+
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <title>Exercicio10php</title> 
+    <link rel="stylesheet" type="text/css" href="estilo.css">
+</head>
+<body>
+<div class="enunciado"> Prencher 20 numeros e submeter ao código</div>
+
+
+   <form action = "index.php" method = "POST">
+    <fieldset class="fundo">
+    <div id="title"><h3>Seleção de Números</h3><br></div>
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br>        
+    <div><input class="digite" name="n2" type="number" placeholder="Digite o numero..."/></div>
+    </br>
+    <div><input class="digite" name="n3" type="number" placeholder="Digite o numero..."/></div>
+    <br>
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
+    <div><input class="digite" name="n1" type="number" placeholder="Digite o numero..."/></div> 
+    <br> 
```

Comment:
- [ ] Arrumar identação
- [ ] Arrumar name que esta tudo igual, ou vc usa [] no final para manter eles iguais ou cada um tera que ter 1 nome.
- [ ] Poderia usar função para escrever esses inputs, lembra, tudo q é codigo repetido pode ser feito por função. Isso ai é um for 20 repetindo varias vezes o mesmo html (não precisa fazer isso, é sugestão, mas vou mostrar na proxima aula)
- [ ] Remover todos os `<br>` e usar margin-top no css

Reply:
preciso rever a aula pra escrever com a função... de cabeça não consegui.